### ISSUE TITLE
Is there a plan to do a soda-thin?

### ISSUE THREAD

**Issue by roachsinai** [Zren/yakuake-sodadark-thintitlebar#3]
I mean not the dark mode?

**Comment by Zren** [comment]
Not really, since I prefer dark themes. The hard part is done though (cropping + layout), so all we really need to do is recolor the pngs. If it's just a 10min project I could look into it later.

Links to Soda Light as I don't have it installed atm in SublimeText.

* https://github.com/buymeasoda/soda-theme
* https://github.com/Miw0/SoDaReloaded-Theme

**Comment by roachsinai** [comment]
OK, thanks!!

**Comment by roachsinai** [comment]
May I ask you how you create these pngs under `tabs` and `title` directories? Software?

**Comment by Zren** [comment]
I used GIMP. `Ctrl+Scroll` to zoom way in till you see the individual pixels in the images.

I opened a screenshot of Sublime Text in GIMP as well. I mostly cropped, cut and pasted from the screenshot IIRC.

If I needed to, I just used the eyedropper tool to select a color from the screenshot, and used the pencil tool to draw the pixels. To do the gradients, I used the eyedropper tool to select the color 1 as the foreground color, and color 2 as the background color. Then I selected a rectangle where I wanted the gradient. Then I used the gradient tool by clicking and holding, then dragging from one end of the selected rectangle to the other do draw the gradient.

**Comment by Zren** [comment]
Eg: For the `title/tabset-background.png` I copied a 1px column.

![2019-01-28___22-43-37](https://user-images.githubusercontent.com/416367/51882927-b275da00-234e-11e9-8c99-e2c8bc063429.png)

Then pasted it 4 times overtop the existing gradient.

![2019-01-28___22-44-52](https://user-images.githubusercontent.com/416367/51882981-fc5ec000-234e-11e9-979e-842ad3c5926d.png)

Which looks like this if we switch to another theme and back:

![2019-01-28___22-46-59](https://user-images.githubusercontent.com/416367/51883019-27491400-234f-11e9-9260-5f53c8af7380.png)

**Comment by roachsinai** [comment]
![screenshot_20190129_115359](https://user-images.githubusercontent.com/9500049/51883122-927cff80-23bc-11e9-9385-4deaab815855.png)

I switch to Arc theme and back, still all black.

**Comment by Zren** [comment]
Did you "export" the file as a `.png`, or did you "save" as an `.xcf`?
